html-forms/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label>
+          Email
+        </label>
+      </div>
+      <input type="email" name="email">
+      <div>
+        <label for="coding is cool">Why is coding cool?</label>
+      </div>
+      <select name="why is coding cool">
+        <option value="Option">Select an option</option>
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+      <div>
+        Favorite Language (choose one)
+      </div>
+      <input type="radio" name="language" value="HTML"/> HTML
+      <input type="radio" name="language" value="CSS" /> CSS
+      <input type="radio" name="language" value="Javascript" /> Javascript
```

Comment:
The text for each of these inputs should be a `label` element that is properly configured to match with the `input`